index.html

```diff
@@ -1 +1,106 @@
+<h1>Email Markup Consortium</h1>
 
+<p>Working to improve the user experience, accessibility, performance,
+consistency, and reliability of email markup</p>
+
+<hr>
+
+<h2>Our Mission</h2>
+
+<p>The Email Markup Consortium (EMC) is a community-led group of industry professionals working to improve the email experience for everyone.</p>
+
+<p>Senders, recipients, and email clients share the common goals of improving:</p>
+
+<ol>
+  <li>user experience</li>
+  <li>accessibility</li>
+  <li>performance</li>
+  <li>consistency</li>
+  <li>reliability</li>
+</ol>
+
+<p>However, currently there is a lack of cooperation in trying to achieve these goals leading to inconsistent results. By working together on agreed approaches we can save everyone time, effort, and money while delivering better results to the recipients.</p>
+
+<p>Recipients should be able to easily access the content of every email sent to them, regardless of their device, email client, assistive technology or internet speed. The EMC collaborates with developers, creation and sending tools, and email clients to drive this vision forward at every step of the email journey.</p>
+
+
+<h2>Quick Links</h2>
+
+<ul>
+  <li>
+    <a href="https://github.com/email-markup-consortium/email-markup-consortium/blob/main/mission.md">
+      Mission statement
+    </a>
+  </li>
+  <li>
+    <a href="https://github.com/email-markup-consortium/email-markup-consortium/blob/main/code-of-conduct.md">
+      Code of conduct
+    </a>
+  </li>
+  <li>
+    <a href="https://github.com/email-markup-consortium/email-markup-consortium/blob/main/compliant-standards.md">
+      Compliant Standards
```

Comment:
Does this need a capital S or should it follow the sentence case pattern established by other items?

Reply:
I don't think a capital 'S' is required here. @M-J-Robbins?

Reply:
I prefer a capital, but happy either way, main thing is to be consistent across all the docs.

Reply:
Let's keep it as is. 

Do we update the other items to match? Or leave them as is?